docs/addons/gpu-operator.md

```diff
@@ -0,0 +1,84 @@
+# NVIDIA GPU-Operator Add-on
+
+Configuring and managing nodes with NVIDIA GPUs requires configuration of multiple software components such as drivers, container runtimes or other libraries which is difficult and error-prone.
+The [gpu-operator](https://github.com/NVIDIA/gpu-operator) automates the management of all NVIDIA software components needed to provision GPU. These components include the NVIDIA drivers (to enable CUDA), Kubernetes device plugin for GPUs, the NVIDIA Container Runtime, automatic node labelling, DCGM based monitoring and others. The GPU Operator allows administrators of Kubernetes clusters to manage GPU nodes just like CPU nodes in the cluster. Instead of provisioning a special OS image for GPU nodes, administrators can rely on a standard OS image for both CPU and GPU nodes and then rely on the GPU Operator to provision the required software components for GPUs.
+
+## Usage
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.GpuOperatorAddon(),
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .version("auto")
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+## Configuration Options
+
+- `createNamespace`: (boolean) If you want CDK to create the namespace for you.
+
+- `values`: Arbitrary values to pass to the chart. Refer to the GPU Operator [Chart Customization Options](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#chart-customization-options) for additional details. It also supports all [standard helm configuration options](https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/docs/addons/index.md#standard-helm-add-on-configuration-options).
```

Comment:
Let's create an explicit type for these values similar to what we did for example for container insights [here](https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/lib/addons/container-insights/values.ts).

Reply:
Created